superset/viz.py

```diff
@@ -1960,7 +1960,7 @@ def get_data(self, df):
             return None
         fd = self.form_data
         label_col = fd.get('mapbox_label')
-        custom_metric = label_col and len(label_col) >= 1
+        custom_metric = bool(label_col) and len(label_col) >= 1
```

Comment:
[optional] we typically don't cast to bool in python, (direct checks on whether a value is truthy are common), but I can see where it's coming from.

Reply:
`custom_metric` is serialized and sent to the client and then used to determine how to cluster the incoming data. Without this change, we get `[]` when using the default point count for aggregation and `true` when using a particular column (e.g. population). The client code is more readable when we send a proper boolean as `custom_metric`. I would probably rename this field to `has_custom_metric` to reflect its type.

Perhaps I should use `if/else` here to produce a boolean for the client to consume?